software/calculations.py

```diff
@@ -41,7 +41,7 @@ def calc_throwingSpeed(self, basket_dist):
                     desired_speed=(basket_dist*t_speeds[i]/t_dist[i])
                     return desired_speed
                 
-                desired_speed=t_speeds[i-1]+(t_speeds[i]-t_speeds[i-1])*percentage
+                desired_speed=t_speeds[i-1]+(t_speeds[i]-t_speeds[i-1])*percentage+100

```

Comment:
I suggest adding a variable you can modify simply up top instead of adding the value on the calculation line. Easier to change